modules/drivers/mcp230/mcp230.js

```diff
@@ -257,34 +222,19 @@ class Pin {
   }
 }
 
-class MCP23008 extends Expander {
-  constructor(dictionary = {}) {
-    super(
-      Object.assign(dictionary, {
-        pins: 8,
-        reg: {
-          IODIR: 0x00,
-          GPPU: 0x06,
-          GPIO: 0x09,
-        }
-      })
-    );
-  }
-}
+class MCP23008 extends Expander {}
+MCP23008.prototype.length = 8;
+MCP23008.prototype.IODIR = 0x00;
+MCP23008.prototype.GPPU = 0x06;
+MCP23008.prototype.GPIO = 0x09;
+Object.freeze(MCP23008.prototype);
+
+class MCP23017 extends Expander {}
+MCP23017.prototype.length = 16;
+MCP23017.prototype.IODIR = 0x00;
+MCP23017.prototype.GPPU = 0x0C;
+MCP23017.prototype.GPIO = 0x12;
+Object.freeze(MCP23017.prototype);
 
-class MCP23017 extends Expander {
-  constructor(dictionary = {}) {
-    super(
-      Object.assign(dictionary, {
-        pins: 16,
-        reg: {
-          IODIR: 0x00,
-          GPPU: 0x0C,
-          GPIO: 0x12,
-        }
-      })
-    );
-  }
-}
```

Comment:
Updates per **Device specific constants** section of https://github.com/Moddable-OpenSource/moddable/pull/21#issuecomment-353265714